camera_calibration/src/camera_calibration/camera_calibrator.py

```diff
@@ -387,3 +451,43 @@ def redraw_stereo(self, drawable):
 
         self._last_display = display
         self.queue_display.put(display)
+
+    def ask_for_calibration(self):
+        if self.c.goodenough:
+            do_calibrate = ask_yes_no_question("Current collected set of samples"
+                                               "is good enough, do you want to calibrate?")
```

Comment:
the string would be "... samplesis good ..." -> there is a whitespaces missing at the linebreak